### ISSUE TITLE
Documentation pages in different browsers

### ISSUE THREAD

**Issue by SlimeDog** [SkriptLang/Skript#1721]
Skript documentation pages display only partial `<div id="content">` content in two of my three browsers. This is true of all documentation pages, not only the page shown below.

### Steps to Reproduce
Open http://skriptlang.github.io/Skript/ in browser. Tested browsers (alphabetical):
- Chrome 43.0.2357.130 (64-bit)
- Firefox 63.0.3 (64-bit)
- Safari 12.0.1 (14606.2.014.1.1)

### Expected Behavior
Pages should look (nearly) identical. Firefox displays correctly; the other two display only partial content.

### Errors / Screenshots
Chrome :: `<div id="content">` and `<div id="nav-contents">` content not displayed
![screen shot 2018-11-29 at 7 04 58 am](https://user-images.githubusercontent.com/17748923/49230704-24f52880-f3a5-11e8-8c49-c2f6fa1c2b43.png)
Firefox :: all content displayed correctly
![screen shot 2018-11-29 at 7 05 53 am](https://user-images.githubusercontent.com/17748923/49230809-5ec62f00-f3a5-11e8-8479-b1675a4924e8.png)
Safari :: topmost `<div id="content">` and `<div id="nav-contents">` content obscured by header
![screen shot 2018-11-29 at 7 08 38 am](https://user-images.githubusercontent.com/17748923/49230915-a1880700-f3a5-11e8-9e47-5578ab02194a.png)

### Server Information
Skript 2.3-beta4 and previous

### Additional Context
MacOS 10.14.1 Mojave

**Comment by TheBentoBox** [comment]
Putting to medium since it seems to make the documentation unusable for some users.

**Comment by SlimeDog** [comment]
Here are the displayed [Triggers](http://skriptlang.github.io/Skript/events.html) pages.

Chrome :: `<div id="content">` content not displayed
![screen shot 2018-11-29 at 8 19 44 am](https://user-images.githubusercontent.com/17748923/49235732-8f12cb00-f3af-11e8-9817-0b88b37dd478.png)
Firefox :: all content displayed correctly
![screen shot 2018-11-29 at 8 17 23 am](https://user-images.githubusercontent.com/17748923/49235639-67236780-f3af-11e8-88fa-15fcda75538e.png)
Safari :: topmost `<div id="content">` and `<div id="nav-contents">` content obscured by header
![screen shot 2018-11-29 at 8 14 39 am](https://user-images.githubusercontent.com/17748923/49235462-0005b300-f3af-11e8-9ecf-b1e691fba4a5.png)

**Comment by ham1255** [comment]
Dude fix you'r browser
![image](https://user-images.githubusercontent.com/34905970/49240628-121f3a00-f41f-11e8-9dde-faf84282887f.png)

**Comment by Nicofisi** [comment]
> Chrome 43.0.2357.130 (64-bit)
The current chrome version is like 70/71/72 (depends on whether the channel is stable, beta, dev or canary) though? :eyes:

![image](https://user-images.githubusercontent.com/12614053/49241054-0cb5f580-f407-11e8-8472-7fc6e50e7a31.png)
![image](https://user-images.githubusercontent.com/12614053/49241067-15a6c700-f407-11e8-8b03-7761bc28f3fd.png)
![image](https://user-images.githubusercontent.com/12614053/49241083-1f302f00-f407-11e8-93cb-3c29689cdc38.png)

**Comment by Efnilite** [comment]
For me it looks the exact same as @Nicofisi's, on Chrome `70.0.3538.110 (Official Build) (64-bit) (cohort: Stable)` on Windows 👀

**Comment by SlimeDog** [comment]
On MacOS:
![screen shot 2018-11-29 at 10 06 54 am](https://user-images.githubusercontent.com/17748923/49242077-95f50a00-f3be-11e8-9a65-eedb0855937d.png)

**Comment by SlimeDog** [comment]
> Dude fix you'r browser
> ![image](https://user-images.githubusercontent.com/34905970/49240628-121f3a00-f41f-11e8-9dde-faf84282887f.png)

Fix in what way?

**Comment by Efnilite** [comment]
How do you fix you'r browser when yours is outdated 🤔

**Comment by Efnilite** [comment]
@ShaneBeee where you at

**Comment by SlimeDog** [comment]
To reiterate... MacOS. All three browsers are latest releases _for MacOS_. Windows versions may or may not be relevant to this issue.

**Comment by bensku** [comment]
Old browser versions are not supported. Please update your web browsers to stay safe in the Internet. PRs to support old browser versions will not be accepted unless they also fix other problems.

Safari support would be nice to have, please send us pull requests. However, I don't have access to Mac so no help with that will be coming from me. That part is also a duplicate of #1598.

**Comment by SlimeDog** [comment]
As noted above, Chrome did not report available updates on MacOS. Updated it manually, and all is well with Chrome. Sucks being Safari. :(

**Comment by ShaneBeee** [comment]
> As noted above, Chrome did not report available updates on MacOS. Updated it manually, and all is well with Chrome. Sucks being Safari. :(

I did a quick for this and sent it to Bensku, just waiting on a reply.
If you would like to test it out [CLICK HERE](https://shanebee.tk/test/expressions.html)
Let me know if that works for you.

**Comment by SlimeDog** [comment]
Yes, ShaneBeee Safari fix works for me. I have no problem using Firefox (Mozilla) browser. I was just reporting the issues.

**Comment by ShaneBeee** [comment]
Awesome 👍 , glad to hear it works for you also. Hopefully they will push this update soon :)

**Comment by SlimeDog** [comment]
I'm in no hurry. I just report 'em. I don't fix 'em. So patience is a virtue.
